### PR TITLE
app-admin/filebeat: bump to 6.3.2

### PR DESCRIPTION
**Pull request by hydrapolic**
Package-Manager: Portage-2.3.45, Repoman-2.3.10

### CODE REVIEW

**Comment by gyakovlev** [comment]
```
QA Notice: Files built without respecting CFLAGS have been detected                                                                                                                         
Please include the following list of files in your report:                                                                                                                                 
  /usr/bin/filebeat
```

Can you please add `QA_FLAGS_IGNORED` ?
golang* eclasses do that, but since this ebuild does not use go related eclass, this needs to be set manually for cleanliness =)

**Comment by monsieurp** [comment]
@hydrapolic: ping.

**Comment by hydrapolic** [comment]
Sure, will do, but currently on vacation.

**Comment by hydrapolic** [comment]
@monsieurp, @gyakovlev pushed an update.

File: app-admin/filebeat/filebeat-6.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline app-admin/filebeat/filebeat-6.3.2.ebuild]: why not 7?

File: app-admin/filebeat/filebeat-6.3.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+inherit eapi7-ver
```
Comment (Zlogene) [inline app-admin/filebeat/filebeat-6.3.2.ebuild]: you do not need this if above

File: app-admin/filebeat/filebeat-6.3.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+pkg_postinst() {
+	if [[ -n "${REPLACING_VERSIONS}" ]]; then
+		elog "Please read the migration guide at:"
+		elog "https://www.elastic.co/guide/en/beats/libbeat/$(ver_cut 1-2)/upgrading.html"
+		elog ""
+	fi
+
+	elog "Example configurations:"
+	elog "${EROOT%/}/usr/share/doc/${PF}/examples"
```
Comment (Zlogene) [inline app-admin/filebeat/filebeat-6.3.2.ebuild]: this too, `${ROOT}` from 7 has a slash in it AFAIR

**Comment by hydrapolic** [comment]
Thanks @Zlogene, bumped to EAPI 7.

**Comment by hydrapolic** [comment]
Thank you guys, appreciated.

**Pull request closed without merging**